src/olympia/reviewers/templates/reviewers/base.html

```diff
@@ -1,6 +1,6 @@
 {% extends "base.html" %}
 
-{% block title %}{{ reviewer_page_title() }}{% endblock %}
+{% block title %} Reviewer Tools – Add-ons for Firefox {% endblock %}
```

Comment:
to make these string translatable they should be wrapped in `{{ _('....') }}`, e.g. {% block title %}{{ _('Reviewer Tools – Add-ons for Firefox') }}{% endblock %}